### PR TITLE
Relaxing some form requirements and errors

### LINKED ISSUE #5877: form action attribute is invalid for method=POST; uncaught console log error. log.js:248
My AMP page validates, but on load has an uncaught error in the JavaScript console saying:

```
Uncaught Error: form action attribute is invalid for method=POST: form#register​​​
    at xa.f.assert (https://cdn.ampproject.org/v0.js:11:342)
    at HTMLDocument.Hf (https://cdn.ampproject.org/v0.js:164:50)
```

I don't undertand the error since the forms action could be used in method=POST (as far I did for years in HTML)
## How do we reproduce the issue?

When visiting http://iberopistacho.com/registro this error should appear on load in the Javascript console.

I'm running ChromeOS Version 54.0.2840.59 (64-bit)
Powered by AMP ⚡ HTML – Version 1477334765771
and using https://cdn.ampproject.org/v0.js

### PR DESCRIPTION
**Pull request by mkhatib**
Related #5607
Related #5670
Closes #5877

### CODE REVIEW

File: src/document-submit.js [hunk 1]
```diff
@@ -103,14 +113,13 @@ export function onDocumentFormSubmit_(e) {
...
 
-  // TODO(#5607): Only require this with method=GET.
   const target = form.getAttribute('target');
-  user().assert(target,
-      'form target attribute is required: %s', form);
-  user().assert(target == '_blank' || target == '_top',
-      'form target=%s is invalid can only be _blank or _top: %s', target, form);
-  if (actionXhr) {
-    checkCorsUrl(actionXhr);
+  if (!target) {
```
Comment (jridgewell) [inline src/document-submit.js]: Please flip the if statement.

Comment (muxin) [inline src/document-submit.js]: Add a test for no target case?

Comment (muxin) [inline src/document-submit.js]: expect it to be `_top`

Comment (mkhatib) [inline src/document-submit.js]: Done

Comment (mkhatib) [inline src/document-submit.js]: Done

**Comment by mkhatib** [comment]
PTAL 👀

**Pull request merged**